File: inventario_python/models/usuario.py
class Usuario:
    def __init__(self):
        self.id = 0
        self.nombre = ""
        self.email = ""
        self.password = None
        self.rol = ""
        self.pregunta_1 = ""
        self.pregunta_2 = ""
        self.pregunta_3 = ""
        self.pregunta_4 = ""
        self.respuesta_1 = ""
        self.respuesta_2 = ""
        self.respuesta_3 = ""
        self.respuesta_4 = ""
        self.intentos_fallidos = 0
        self.bloqueado_hasta = None
        self.ultimo_login = None
        self.permisos = ""

File: inventario_visual/Models/Usuario.vb
Namespace inventario_visual
    Public Class Usuario
        Public Property Id As Long
        Public Property Nombre As String
        Public Property Email As String
        Public Property PasswordHash As String
        Public Property Rol As String
        Public Property Permisos As String
        Public Property Pregunta1 As String
        Public Property Respuesta1 As String
        Public Property Pregunta2 As String
        Public Property Respuesta2 As String
        Public Property Pregunta3 As String
        Public Property Respuesta3 As String
        Public Property Pregunta4 As String
        Public Property Respuesta4 As String
        Public Property IntentosFallidos As Integer
        Public Property BloqueadoHasta As DateTime?
        Public Property UltimoLogin As DateTime?
    End Class
End Namespace
